src/store/index.js

```diff
@@ -1,17 +1,43 @@
-import Vue from 'vue'
-import Vuex from 'vuex'
+import Vue from "vue";
+import Vuex from "vuex";
+import axios from "axios";
 
-Vue.use(Vuex)
+Vue.use(Vuex);
 
 export default new Vuex.Store({
   state: {
+    movies: [],
+    searchQuery: "",
   },
   getters: {
+    getMovies: (state) => state.movies,
   },
   mutations: {
+    SET_MOVIES(state, movies) {
+      state.movies = movies;
+    },
+    SET_SEARCH_VALUE(state, query) {
+      state.searchQuery = query;
+    },
```

Comment:
You don't commit this mutation

Reply:
Changed the logic of application. Now I commit.